mqtt/mqtt-bridge/src/persist/waking_state/ring_buffer/mod.rs

```diff
@@ -952,6 +966,59 @@ mod tests {
         }
     }
 
+    #[test]
+    fn it_inserts_ok_after_leftover_and_wrap_around_is_smaller() {
+        let mut rb = TestRingBuffer::default();
+        let publication = Publication {
+            topic_name: "test".to_owned(),
+            qos: QoS::AtMostOnce,
+            retain: true,
+            payload: Bytes::new(),
+        };
+
+        let block_size = *SERIALIZED_BLOCK_SIZE;
+
+        let result = bincode::serialize(&publication);
+        assert_matches!(result, Ok(_));
```

Comment:
looks a bit odd, I think we just do `.unwrap()` or better `.expect("...")` in tests...

Reply:
Should I fix all of these throughout the tests in this PR or just these?

Reply:
I can also open up another pr to get all of these as well if that is preferred :)

Reply:
Went ahead and changed these for this test, then if we want to change the others later we can :)